datumaro/cli/commands/log.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import argparse
+
+from ..util.project import load_project
+
+
+def build_parser(parser_ctor=argparse.ArgumentParser):
+    parser = parser_ctor(description="Prints project history.")
+
+    parser.add_argument('-n', '--count', default=10, type=int,
```

Comment:
I would suggest to rename it to `--max-count`, for consistency with Git.

Reply:
Done